resources/views/components/game/top-achievers/most-points.blade.php

```diff
@@ -0,0 +1,53 @@
+@props([
+    'highestPointEarners' => [], // Collection<PlayerGame>
+    'isEvent' => false,
+])
+
+@php
+
+use App\Models\User;
+
+@endphp
+
+<div class="component">
+    <h2 class="text-h3">Most Points Earned</h2>
+
+    <div class="max-h-[980px] overflow-y-auto">
+        @if (empty($highestPointEarners))
+        <tr colspan="3">No points have been earned yet.</tr>
```

Comment:
This is inside of a table row, but outside of a table -- technically invalid HTML. Might be best to change this to a `<p>` tag?

Reply:
I had it in the table first, but then moved it out. Fixed.